_tools/check_doc_imports.ts

```diff
@@ -26,6 +26,11 @@ const RX_CODE_BLOCK = /`{3}([\w]*)\n([\S\s]+?)\n`{3}/gm;
 let shouldFail = false;
 let countChecked = 0;
 
+const allowedUrlPrefixes = [
+  "https://deno.land/std@$STD_VERSION/",
+  "https://html.spec.whatwg.org/",
```

Comment:
Is it OK to add a special case for this? Rationale — it's a JSON version of [a spec page provided by WHATWG](https://html.spec.whatwg.org/multipage/named-characters.html), which is the authoritative reference for named HTML entities and which states:

> _This list is static and [will not be expanded or changed in the future](https://github.com/whatwg/html/blob/main/FAQ.md#html-should-add-more-named-character-references)._

The alternative to importing directly from the WHATWG-provided JSON would be duplicating that JSON file within `std`.

Reply:
I'd prefer to duplicate the file in std with some note about the original. The spec doesn't seem guaranteeing the url and the shape of that JSON file.

Reply:
Fixed — one other advantage of importing internally is it doesn't have to use the same structure, which is more verbose than it needs to be for our purposes.